README.md

```diff
@@ -211,34 +219,88 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
```

Comment:
There are several different possible outputs. (doesn't tie us down to a specific number if outputs are added/removed)